views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+          States of America</a> has the <em>House of Representatives</em> and
+          the <em>Senate</em>.
+        </p>
+        <p>
+          Furthermore, we split the data for those down into
+          <strong>terms</strong>. Like legislatures themselves, the way this
+          works may vary from country to country. Terms have a start and end
+          date, and the data therefore describes the membership of the
+          legislation during that period. In most democracies, new terms begin
+          after every national election.
+        </p>
+        <p>
+          This is pragmatic because we know online services and sites are
+          likely to be concerned with the country’s <em>current</em>
+          legislature, which means the data for politicians in the current
+          term. Often — but not always — this is also the data we’re most
+          likely to be able to source online, so if you see a country only has
+          one term in its data, it may well be the current one. We’re always
+          interested in adding historic data too (that is, data for previous
+          terms), so if you know where we can get it for your country, please
+          let us know. Similarly, if we are lacking the current term (maybe
+          there’s just been an election?), please let us know — it might be
+          that our source has been updated and we haven’t pulled down the
+          latest changes.
+        </p>
+        <h2>
+          Politicians
+        </h2>
+        <p>
+          <strong>The richness of the data we have will depend on the source or
+          sources from which we’re getting it.</strong> We use a wide variety
+          of sources, some official and many more that are not. Indeed, the
+          EveryPolitician project exists partly because in many places it’s
+          still depressingly hard to get even basic information about the
+          current legislature.
+        </p>
+        <p>
+          But the absolute minimum data we need for a politician to be in
+          EveryPolitician is their <strong>name</strong> (by implication, we
+          also have their membership of the legislature and its term). Unless
+          the system they are in does not use such mechanisms, we also try to
+          have the <strong>area</strong> they represent, and the
+          <strong>group</strong> (perhaps a party or faction) they belong to.
+          There may also be start and end dates if we know they did not serve
+          the full term.
+        </p>
+        <a name="about-names"></a>
+        <h3>
+          About names
+        </h3>
+        <p>
+          Names are a key part of any data set based around people. In the
+          basic CSV, the <code>name</code> will be the common name that we’ve
+          got from the source, but we’re aware that names are not quite that
+          simple. We often have this name data:
```

Comment:
"Often" is perhaps a little strong. Perhaps something like "So we sometimes have separate fields for"?